scalastyle-config.xml

```diff
@@ -3,7 +3,7 @@
  <check level="error" class="org.scalastyle.file.FileTabChecker" enabled="true"></check>
  <check level="error" class="org.scalastyle.file.FileLengthChecker" enabled="true">
   <parameters>
-   <parameter name="maxFileLength"><![CDATA[800]]></parameter>
+   <parameter name="maxFileLength"><![CDATA[1000]]></parameter>
```

Comment:
Hmm, I think this scalastyle config is already aligned with other Spotify Scala projects. I don't think we should change this and should instead use ignore or refactor when absolutely necessary

Reply:
good point, reverting this and adding ignore file length check